react-carousel-component/nimmys-app/src/App.js

```diff
@@ -0,0 +1,40 @@
+import './App.css';
+import Container from './Container';
+
+const players = [
+  {
+    key: 0,
+    name: 'Kobe',
+    img: './players/youngKob.jpeg',
```

Comment:
I think these should be absolute paths. Unfortunately, HTTP resource paths are handled different than source code paths! An absolute path will be resolved relative to the `public` folder while a relative path will be resolved relative to whatever URL is currently in the address bar. In this app, the URL in the address bar is always `/`, so the relative path works, but in a larger app this won't always be the case.